pkg/serverless/logs/testdata/invalid_log_no_type.json

```
@@ -0,0 +1 @@
+{ "record": "sample extension log" }
```

Just to avoid confusion, since this isn't actually a valid extension log.

```suggestion
{ "record": "invalid log without a type" }
```